packages/onboarding-ui/src/pages/AdminInfoPage/AdminInfoPage.tsx

```diff
@@ -7,9 +7,12 @@ import AdminInfoForm from '../../forms/AdminInfoForm';
 import type { AdminInfoPayload } from '../../forms/AdminInfoForm/AdminInfoForm';
 
 type AdminInfoPageProps = {
+  title?: string;
+  description?: string;
```

Comment:
For more flexibility:

```suggestion
  title?: ReactNode;
  description?: ReactNode;
```

Reply:
Sounds good!